### PR TITLE
Modified documentation and added doc for undocced functions.

### PR DESCRIPTION
**Pull request by shubhamjain0594**
- Added internal page links for easy navigation
- Added documentation for undocced functions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -179,13 +179,15 @@ vis._send({'data': [trace], 'layout': layout, 'win': 'mywin'})
 ```
 
 ### Others
-- `vis.close`    : close a window by id
-- `vis.win_exists`: check if a window already exists by id
-- `vis.check_connection`: check if the server is connected
+- [`vis.close`](#visclose)    : close a window by id
+- [`vis.win_exists`](#viswinexists) : check if a window already exists by id
```
Comment (JackUrb) [inline README.md]: #viswin_exists and #vischeck_connection are the correct links

File: README.md [hunk 2]
```diff
@@ -419,10 +424,26 @@ The following `opts` are generic in the sense that they are the same for all vis
...
+#### vis.close
+
+This function closes a specific window. It takes input window id `win` and environment id `eid`. Use `win` as `None` to close all windows in an environment.
+
+#### vis.win_exists
+
+This function returns a bool indicating whether or not a window `win` exists on the server already. Returns None if something went wrong. 
+
+Optional arguments:
+- `env`: Environment in which window is to be searched. Default is `None`.
```
Comment (JackUrb) [inline README.md]: Small description preference nit:
Environment to search for the window in. Default is `None`.

File: README.md (hunk 1 above)
Comment (shubhamjain0594) [reply README.md]: Ohh, that worked on [stackedit](https://stackedit.io/). Addressed in the commit.

File: README.md (hunk 2 above)
Comment (shubhamjain0594) [reply README.md]: Addressed.

**Comment by JackUrb** [comment]
Cool I'm going to pull this in. Thanks for the improvements to our docs!

**Comment by shubhamjain0594** [comment]
@JackUrb Thank you.

**Pull request closed without merging**